powerVCF.psd1

```diff
@@ -8,41 +8,41 @@
 
 @{
 
-# Script module or binary module file associated with this manifest.
-RootModule = 'PowerVCF.psm1'
+    # Script module or binary module file associated with this manifest.
+    RootModule        = 'PowerVCF.psm1'
```

Comment:
Sorry what value are you trying to add here, the file was auto generated by the PowerShell tools

Reply:
Yes, i known but we don't regenerate after...